tools/releaseBuild/azureDevOps/templates/shouldSign.yml

```diff
@@ -1,9 +1,12 @@
 steps:
 - powershell: |
     $shouldSign = $true
-    if($env:BUILD_REASON -ne 'Manual')
```

Comment:
Does this mean that builds that are not manual and are not daily will now be signed with the standard Authenticode cert?

Reply:
I believe we do not have any builds for this pipeline that are not manual and not daily. The builds we have are:

1. Daily build - kicked off automatically so not manual. -> signed 
2. Manual kicked off for release - so not daily
3. Manually kicked off for testing - so not daily

The pipeline also has an option to skip signing using the pipeline schedule time parameter.

Reply:
No automated on-push branch build?

Reply:
on-push is disabled for the pipeline.